react/ProductSummaryCustom.tsx

```diff
@@ -162,44 +161,49 @@ const ProductSummaryCustom = ({
           </section>
         </ProductPriceSimulationWrapper>
       </ProductContextProvider>
-    </ProductSummaryContext.Provider>
+    </LocalProductSummaryContext.Provider>
   )
 }
 
-ProductSummaryCustom.propTypes = {
-  /** Product that owns the informations */
-  product: productShape,
+interface Props {
```

Comment:
Why not also add `classes` in this component? I've just noticed the Wrapper below, you would need to include it there too :)